tests/appsec/test_automated_login_events.py

```diff
@@ -0,0 +1,154 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2022 Datadog, Inc.
+
+from utils import weblog, interfaces, context, missing_feature, released, scenarios, coverage, rfc
+
+
+@rfc("https://docs.google.com/document/d/1-trUpphvyZY7k5ldjhW-MgqWl0xOm7AMEQDJEAZ63_Q/edit#heading=h.8d3o7vtyu1y1")
+@coverage.good
+@released(cpp="?", golang="?", java="?", nodejs="", dotnet="?", php="?", ruby="?")
```

Comment:
python is missing, and node is empty. If you want to wait for the release, you need to add a `?` in nodejs : 

```suggestion
@released(cpp="?", golang="?", java="?", nodejs="?", dotnet="?", php="?", python="?", ruby="?")
```